### ISSUE TITLE
Graceful exit when given incorrect --check input

### ISSUE THREAD

**Issue by mahrud** [Macaulay2/M2#3488]
This is very niche, but it's a pain when loading startup fails and you don't even have `exit`:
```m2
mahrud@noether:~$ M2 --check 0
/tmp/macaulay2-20240718-40801-p223zn/M2-release-1.24.05/M2/Macaulay2/m2/startup.m2.in:522:16:(0):[1]: --check 0: expected 1, 2, or 3
/tmp/macaulay2-20240718-40801-p223zn/M2-release-1.24.05/M2/Macaulay2/m2/startup.m2.in:522:16:(0):[1]: --entering debugger (type help to see debugger commands)

i1 : exit

o1 = <<a function closure>>

i2 : exit 0
```

**Comment by d-torrance** [comment]
Should we just call "exit 1" or something automatically when this happens?

**Comment by mahrud** [comment]
Sure, that seems reasonable. Also if the checks failed I think.

**Comment by d-torrance** [comment]
Closed by #3496
